Homework2/src/AddFoodToCartServlet.java

```diff
@@ -0,0 +1,37 @@
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/menu/add")
+public class AddFoodToCartServlet extends HttpServlet {
+
+	public void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+
+		int id = Integer.parseInt(request.getParameter("id"));
+		List<FoodEntry> entries = (List<FoodEntry>) getServletContext().getAttribute("entries");
+		FoodEntry leEntry = null;
+		for (FoodEntry entry: entries) {
+			if (entry.getId() == id) {
+				leEntry = entry;
+			}
+		}
+		response.setContentType("text/html");
+		PrintWriter out = response.getWriter();
```

Comment:
If you are already doing redirect, you don't need to set contentType to "text/html" nor getting this print writer variable.